Content/install/liquibase-requirements.htm

```diff
@@ -21,8 +21,8 @@ <h2>Software</h2>
         <ul>
             <ul>
                 <li>If you use the <MadCap:variable name="General.Liquibase" /> Installer, Java is included automatically.</li>
-                <li>If you install <MadCap:variable name="General.Liquibase" /> manually, you must also install Java, which is available from the following sources:<ul><li><a href="https://www.oracle.com/java/technologies/javase-downloads.html">OpenJDK by Oracle</a> – Oracle provides the latest OpenJDK release under the open-source GPL License. If you use the <MadCap:variable name="General.Liquibase" /> Installer, this version of Java is used during installation.</li><li><a href="https://www.java.com/en/download/manual.jsp">Java by Oracle</a> – Oracle provides the latest Java release for all operating systems.</li></ul></li>
+                <li>If you install <MadCap:variable name="General.Liquibase" /> manually, you must also install Java, which is available from <a href="https://adoptium.net/temurin/releases">Adoptium</a>. If you use the <MadCap:variable name="General.Liquibase" /> Installer, this version of Java is used during installation.</li>
```

Comment:
It's good to direct people to use the same version Liquibase installs and that we test with when using the installed Liquibase - thank you!

PS Also, this is much, much cleaner. If I'm reading documentation that tells me several different locations for the same(ish) file, I worry I'm going to somehow pick the "wrong" one.